go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/treeverse/lakefs
 
-go 1.17
+go 1.18
```

Comment:
Are we migrating to 1.18 as part of this change?
It was just released and I think we should wait some more before using it..